lib/client-assets.php

```diff
@@ -783,12 +783,12 @@ function gutenberg_editor_scripts_and_styles( $hook ) {
 	$gutenberg_theme_support = get_theme_support( 'gutenberg' );
 	$color_palette = gutenberg_color_palette();
 
-	if ( $gutenberg_theme_support && $gutenberg_theme_support[0]['colors'] ) {
+	if ( $gutenberg_theme_support && ! empty( $gutenberg_theme_support[0]['colors'] ) ) {
```

Comment:
I'm wondering if `empty` isn't powerful enough to make it possible to simplify it to:
```php
if ( ! empty( $gutenberg_theme_support[0]['colors'] ) ) {
```

Reply:
Sorry, I'm a little lost. Can you explain further?

Reply:
> Determine whether a variable is considered to be empty. A variable is considered empty if it does not exist or if its value equals FALSE. empty() does not generate a warning if the variable does not exist.

You don't need to check if `$gutenberg_theme_support` is truthy, because `empty` does that, too.

Reply:
Just tested with the following code:

```php
function mytheme_setup_theme_supported_features() {
	add_theme_support( 'gutenberg', null );
}

add_action( 'after_setup_theme', 'mytheme_setup_theme_supported_features' );
```

Reply:
Yeah that's right, should I update the PR with this?

Reply:
If you like it, then go for it. Totally up to you :)

Reply:
Updated :)